src/dlgTriggerEditor.cpp

```diff
@@ -6766,6 +6766,55 @@ void dlgTriggerEditor::changeView(EditorViewType view)
     checkBox_displayAllVariables->setVisible(view == EditorViewType::cmVarsView);
 
     mpExportAction->setEnabled(view != EditorViewType::cmVarsView);
+
+    switch (mCurrentView) {
+    case EditorViewType::cmTriggerView:
+        mAddItem->setIconText(tr("Add Trigger"));
+        mAddGroup->setIconText(tr("Add Trigger Group"));
+        mDeleteItem->setIconText(tr("Delete Trigger"));
+        mSaveItem->setIconText(tr("Save Trigger"));
+        break;
+    case EditorViewType::cmTimerView:
+        mAddItem->setIconText(tr("Add Timer"));
+        mAddGroup->setIconText(tr("Add Timer Group"));
+        mDeleteItem->setIconText(tr("Delete Timer"));
+        mSaveItem->setIconText(tr("Save Timer"));
+        break;
+    case EditorViewType::cmAliasView:
+        mAddItem->setIconText(tr("Add Alias"));
+        mAddGroup->setIconText(tr("Add Alias Group"));
+        mDeleteItem->setIconText(tr("Delete Alias"));
+        mSaveItem->setIconText(tr("Save Alias"));
+        break;
+    case EditorViewType::cmScriptView:
+        mAddItem->setIconText(tr("Add Script"));
+        mAddGroup->setIconText(tr("Add Script Group"));
+        mDeleteItem->setIconText(tr("Delete Script"));
+        mSaveItem->setIconText(tr("Save Script"));
+        break;
+    case EditorViewType::cmActionView:
+        mAddItem->setIconText(tr("Add Button"));
+        mAddGroup->setIconText(tr("Add Button Group"));
```

Comment:
```suggestion
        mAddGroup->setIconText(tr("Add Toolbar (top level)/ Menu (lower levels)"));
```

Reply:
That is way too long for a button

Reply:
:thinking: Yeah, but "Button Group" is confusing when, for that particular type of element, it produces two different things depending on where in the `TTreeWidget` hierarchy it is used...

Reply:
The confusion is actually result of the confusing software design there, and really confuses players when you need to keep explaining it at every place that handles a button group.